Tasks/AutomatedAnalysisV0/task.json

```diff
@@ -0,0 +1,468 @@
": \"TEMPLATE_SUBSTITUTIONS\", \"secureValue\": \"{{{ #stringReplace '\"' '\\\"' templateSubstitutions }}}\" }, { \"name\": \"ARE_METRICS_CRITICAL\", \"value\": \"{{ areMetricsCritical }}\" }, { \"name\": \"MUST_HAVE_DATA\", \"value\": \"{{ mustHaveData }}\" }, { \"name\": \"ANALYSIS_TYPE\", \"value\": \"{{ analysisType }}\" }, { \"name\": \"WINDOW_TYPE\", \"value\": \"{{ windowType }}\" }, { \"name\": \"DELAY\", \"value\": \"{{ delay }}\" }, { \"name\": \"LIFETIME\", \"value\": \"{{ lifetime }}\" }, { \"name\": \"ANALYSIS_BEGIN_TIME\", \"value\": \"{{ analysisBeginTime }}\" }, { \"name\": \"ANALYSIS_END_TIME\", \"value\": \"{{ analysisEndTime }}\" }, { \"name\": \"INTERVAL\", \"value\": \"{{ interval }}\" }, { \"name\": \"PREPROCESSOR\", \"value\": \"{{ preprocessor }}\" },  {\"name\": \"NAN_STRATEGY\", \"value\": \"{{ nanStrategy }}\" }, { \"name\": \"CLASSIFIER\", \"value\": \"{{ classifier }}\" }, { \"name\": \"METRIC_GROUPS\", \"value\": \"{{ metricGroups }}\" },  { \"name\": \"GROUP_WEIGHTS\", \"value\": \"{{ groupWeights }}\" },  { \"name\": \"DIRECTION\", \"value\": \"{{ direction }}\" }, { \"name\": \"CONFIDENCE_LEVEL\", \"value\": \"{{ confidenceLevel }}\" }, { \"name\": \"ALLOWED_INCREASE\", \"value\": \"{{ allowedIncrease }}\" }, { \"name\": \"ALLOWED_DECREASE\", \"value\": \"{{ allowedDecrease }}\" }, { \"name\": \"CRITICAL_INCREASE\", \"value\": \"{{ criticalIncrease }}\" }, { \"name\": \"CRITICAL_DECREASE\", \"value\": \"{{ criticalDecrease }}\" }, { \"name\": \"MARGINAL_THRESHOLD\", \"value\": \"{{ marginalThreshold }}\" }, { \"name\": \"PASS_THRESHOLD\", \"value\": \"{{ passThreshold }}\" }, {\"name\":\"LOG_LEVEL\", \"value\":\"$(logLevel)\"}, {\"name\":\"LOG_OUTPUT\", \"value\":\"FILE\"}], \"image\":\"ansheno/aa:v1\",\"ports\":[{\"port\": 8080}], \"volumeMounts\":[{\"name\":\"varlog\", \"mountPath\":\"/usr/src/aa/logs\", \"readOnly\": \"false\"}], \"resources\": {\"requests\": {\"cpu\": 1,\"memoryInGB\": 0.5}}}}, {\"name\":\"aa-sidecar\",\"properties\":{\"volumeMounts\":[{\"name\":\"varlog\", \"mountPath\":\"/usr/src/aa/logs\", \"readOnly\": \"false\"}], \"resources\": {\"requests\": {\"cpu\": 1,\"memoryInGB\": 0.5}}, \"command\":[], \"environmentVariables\":[{ \"name\":\"LOGS_PROVIDER\", \"value\":\"FILE\"}, {\"name\":\"LOG_FILE_PATH\", \"value\":\"/usr/src/aa/logs/out.log\"}, {\"name\":\"TASK_PROPERTIES\", \"value\":\"{\\\"Content-Type\\\":\\\"application/json\\\", \\\"PlanUrl\\\": \\\"$(system.CollectionUri)\\\", \\\"ProjectId\\\": \\\"$(system.TeamProjectId)\\\", \\\"HubName\\\": \\\"release\\\", \\\"PlanId\\\": \\\"$(system.PlanId)\\\", \\\"JobId\\\": \\\"$(system.JobId)\\\", \\\"TimelineId\\\": \\\"$(system.TimelineId)\\\", \\\"TaskInstanceId\\\": \\\"$(system.TaskInstanceId)\\\", \\\"TaskInstanceName\\\": \\\"$(system.TaskInstanceName)\\\", \\\"AuthToken\\\": \\\"$(system.AccessToken)\\\"}\"}], \"image\":\"ansheno/azdevlogger:n5\", \"ports\":[{\"port\": 443}]}}]},\"type\":\"Microsoft.ContainerInstance/containerGroups\"}",
```

Comment:
Make the mount value in task.json as `/usr/src/logs` and `LOG_FILE_PATH` as /usr/src/logs/out.log 

The WORKDIR has been changed in Automated Analysis repo